### ISSUE TITLE
Pi-hole FTL v5.14, Web v5.11 and Core v5.9 released

### ISSUE THREAD

**Issue by town3r** [arevindh/pihole-speedtest#105]
https://pi-hole.net/blog/2022/02/12/pi-hole-ftl-v5-14-web-v5-11-and-core-v5-9-released/

**Comment by corasaniti** [comment]
when you plan to upgrade to versions
Pi-hole v5.9 FTL v5.14 Web Interface v5.11
Thanks in advance

**Comment by corasaniti** [comment]
Can I perform the pihole-up update even inside a container? 
If I then remove and delete the container do I lose the changes that have not persisted in the docker image?
Thanks

**Comment by ChrisWNY** [comment]
Docker images/builds would have to be maintained and updated as the software components themselves are updated. So if there's a new version of software available that's part of your docker image you'd have to also wait on the maintainer to update that image file.

**Comment by arevindh** [comment]
@town3r @ChrisWNY @corasaniti 
Will update tomorrow.

**Comment by arevindh** [comment]
@town3r @ChrisWNY Updated, please let me know after testing

**Comment by arevindh** [comment]
@corasaniti docker image also updated, please check and confirm

**Comment by town3r** [comment]
> @town3r @ChrisWNY Updated, please let me know after testing

Thanks for the quick turn around. 

Looks good so far!

Now if only the dashboard graph for Speedtest matched the others in the `Star Trek LCARS theme (dark)` theme. ;-)
![image](https://user-images.githubusercontent.com/12396203/154284683-97949174-03b6-473e-81f0-6b0be74b5e14.png)

**Comment by arevindh** [comment]
@corasaniti docker image has some issues , please don't update now.

**Comment by arevindh** [comment]
> Star Trek LCARS theme (dark)

Let me try what I can do this weekend.

**Comment by corasaniti** [comment]
@arevindh Docker container works great. You have been very kind
Thank you :-)

**Comment by town3r** [comment]
> @town3r can you close this and create that create as a separate issue, It is easier to track .

done! https://github.com/arevindh/pihole-speedtest/issues/106
